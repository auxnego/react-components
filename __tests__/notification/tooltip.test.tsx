import React from "react";
import {create} from "react-test-renderer";

import Tooltip from "../../notification/tooltip";

it("No open tooltip", () => {

  expect(create(
    <Tooltip open={false} >Tooltip Base</Tooltip>
  ).toJSON()).toMatchSnapshot();

});

it("opened tooltip", () => {

  expect(create(
    <Tooltip open={true} >Tooltip Base</Tooltip>
  ).toJSON()).toMatchSnapshot();

});

it("opened tooltip with setClass , setEl and setProps", () => {

  expect(create(
    <Tooltip open={true} 
     setClass={{root:"custom"}}
     setEl={{arrow:"span"}}
    >Tooltip Base</Tooltip>
  ).toJSON()).toMatchSnapshot();

});
