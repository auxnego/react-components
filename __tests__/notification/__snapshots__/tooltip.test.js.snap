// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No open tooltip 1`] = `null`;

exports[`opened tooltip 1`] = `
<div
  className="tooltip__root"
>
  <div
    className="tooltip__arrow"
  />
  <div
    className="tooltip__content"
  >
    Tooltip Base
  </div>
</div>
`;

exports[`opened tooltip with setClass , setEl and setProps 1`] = `
<div
  className="tooltip__root"
>
  <div
    className="tooltip__arrow"
  />
  <div
    className="tooltip__content"
  >
    Tooltip Base
  </div>
</div>
`;
